ql/models/shortrate/onefactormodels/gsr.cpp

```diff
@@ -92,6 +92,12 @@ Gsr::Gsr(const Handle<YieldTermStructure> &termStructure,
     initialize(T);
 }
 
+void Gsr::update() { 
```

Comment:
Is this (and the other changes below) part of the work on optimizers, or something else?